packages/itwin/tree-widget/src/tree-widget-react/components/trees/common/internal/Tooltip.ts

```diff
@@ -26,26 +25,13 @@ export function createVisibilityStatus(status: Visibility | "disabled", { useToo
   return {
     state: status === "disabled" ? "hidden" : status,
     isDisabled: status === "disabled",
-    tooltip: useTooltip === false ? undefined : createTooltip(status, useTooltip),
+    tooltip: useTooltip === false ? undefined : createTooltip(status),
```

Comment:
The `useTooltip` prop is optional, but you're checking it for `false` here. So it looks like callers must specify the prop if they want to omit the tooltip. But everywhere I look the prop is not specified, which means the tooltip is always created.

Reply:
Do we even need to have `tooltip` on `VisibilityStatus`? With this implementation it could be determined by state when rendering action.